Changed ex2

chrismeim opened the pull request.

chrismeim committed:
Changed ex2

teohaik commented on Excercise 2.py:
```diff
@@ -1,9 +1,10 @@
-degrees = int(input("Give me the temperature"))
+degrees = int(input("Give me the temperature "))
 
 def converter(x, z):
```
Variable names should be meaningfull and not single letters.
Eg Use: `option, temp` instead of `x,z`

teohaik replied:
function names should be meaningfull also. 
What is this converter? What does it convert?

chrismeim committed:
Test

chrismeim committed:
Changing Excercise Names

chrismeim committed:
Changes excercise 2

chrismeim committed:
Change Ex3-6

chrismeim committed:
Change Ex 10-11

chrismeim committed:
Create Excercise 14.py

Calculates the number of characters, numbers and spaces in a string

chrismeim committed:
Create Excercise 15.py

Checks if a password meets certain criteria and tells the user if the password is wrong or right

chrismeim committed:
Create Excercise 16.py

Creates a list where numbers are included if all of their digits are even.

chrismeim committed:
Uploaded Ex17, corrected ex15

chrismeim committed:
Excercise 18

chrismeim committed:
Excercise 19

chrismeim committed:
Excercise 31

chrismeim committed:
Ex 32

chrismeim committed:
Excercise 33

chrismeim committed:
Excercise 34

chrismeim committed:
Excercise 36

chrismeim committed:
Excercise 42, sum and average of n inputs from user

chrismeim committed:
Merge branch 'Working_with_Teo_ex2' of https://github.com/chrismeim/Repository-1.1 into Working_with_Teo_ex2

chrismeim committed:
Problem 28 leet

chrismeim committed:
minor changes

chrismeim committed:
Problem 35 leet

chrismeim committed:
Exc 66 leet

chrismeim committed:
Problem70leet

chrismeim committed:
Problem 83 Remove Duplicates

chrismeim committed:
Problem 100 leet
